source/image-handler/image-request.js

```diff
@@ -46,6 +46,8 @@ class ImageRequest {
         const request = s3.getObject(imageLocation).promise();
         try {
             const originalImage = await request;
+            this.Expires = originalImage.Expires;
+            this.LastModified = originalImage.LastModified;
```

Comment:
I've tried this fix and the s3.getObject() call returns an invalid date format (e.g. 2019-11-06T21:36:00.000z). Added the following to change into a valid HTTP header format.

`const formattedLastModified = new Date(originalImage.LastModified).toUTCString();`

Reply:
Thanks. I noticed this, but hadn't had time to investigate it. That fix is in the pull request now.